PizzaPyWebApp/templates/partials/landing/cta.html

```diff
@@ -0,0 +1,34 @@
+<div class="relative flex flex-col items-center justify-center w-full min-h-screen">
+    <img src="https://via.placeholder.com/1500x800" alt="Background Image"
+        class="absolute inset-0 z-0 object-cover w-full h-full opacity-75">
+    <div
+        class="relative z-10 flex flex-col items-center justify-center w-full gap-10 p-6 bg-opacity-50 sm:gap-20 md:p-28 dark:bg-gray-900 backdrop-blur">
+        <div
+            class="relative z-10 flex flex-col items-center justify-center w-full max-w-screen-lg p-6 text-center text-white sm:p-8 md:p-10 lg:p-12">
+            <h1 class="text-4xl font-bold sm:text-5xl md:text-6xl">
+                What are you waiting for? <br>
+                <span class="text-yellow-500">Join us now.</span>
+            </h1>
+            <p class="mt-4 text-lg sm:text-xl">
+                Begin your journey into the wonderful world of Cebu, Python, and Pizza!
+            </p>
+            <div
+                class="flex flex-col items-center justify-center w-full gap-4 px-4 mt-8 sm:flex-row sm:w-3/4 md:w-2/3 sm:px-0">
+                <button
```

Comment:
The button background color should be consistent with the other buttons: `#5248FF`.